generator/templates/default/src/plugins/vuetify.js

```diff
@@ -12,7 +25,7 @@ import {
   VToolbar,
   transitions
 } from 'vuetify'
-import '../node_modules/vuetify/src/stylus/app.styl'
+import 'vuetify/src/stylus/app.styl'

```

Comment:
that fix is already in #14, how about we merge #14 and then remove it from here?

Reply:
This is something I did like a month ago but forgot to commit.